apps/pie-docs/src/assets/js/search.js

```diff
@@ -0,0 +1,81 @@
+// eslint-disable-next-line
+docsearch({
+    // public algolia keys (can be exposed)
+    appId: 'L21G25CLL1',
+    apiKey: '75fcb9c4b333a7a004ae63e6a736a81e',
+    indexName: 'pie',
+    container: '#docsearch',
+});
+
+document.querySelector('.DocSearch').onclick = () => {
+    const startScreenOptions = [
+        {
+            category: {
+                name: 'All About PIE',
+                icon: '/assets/img/search/pie.svg',
+            },
+            items: [
+                { name: 'What is PIE', href: '/all-about-pie/what-is-pie/' },
+                { name: 'Our Vision', href: '/all-about-pie/our-vision/' },
+                { name: 'Roadmap', href: '/all-about-pie/roadmap/' },
+            ],
+        },
+        {
+            category: {
+                name: 'Designers',
+                icon: '/assets/img/search/bulb-lightning.svg',
```

Comment:
do you think we can use our icons library here? 
they all should be available.

Reply:
I thought that too but `search.js` is used inside a <script> tag so I'm unable to use require and if I import '@justeattakeaway/pie-icons' it asks for a relative path 🤷 not quite sure how else to tackle it. do you know a way? 🤞

Reply:
agreed to look at this separately 😄